initialization/initialization_test.go

```diff
@@ -49,9 +49,9 @@ func TestInitialize(t *testing.T) {
 	r.NoError(err)
 	init.SetLogger(log)
 
-	eg, _ := errgroup.WithContext(context.Background())
+	var eg errgroup.Group
+	eg.Go(init.Initialize)
```

Comment:
Why does it need to run asynchronously?

Reply:
This is already in preparation of `init.Initialize()` becoming a blocking call in the future as outlined in https://github.com/spacemeshos/post/issues/78

Reply:
Then, IMO, it should be done with the changes for #78.